src/components/ui/layout/sidebar.tsx

```
@@ -133,6 +133,62 @@ export function AppSidebar({ ...props }: React.ComponentProps<typeof Sidebar>) {
+            
+            console.log(newAnalyzeNav)
+            setNavContent((prev) =>
+              prev.map((i) =>
+                i.id === "analyze" ? { ...i, items: newAnalyzeNav.items } : i,
+              ),
+            );
+          });
+      }, [user]);
+
+      React.useEffect(() => {
+        const handler = (e: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)